### DISCUSSION TITLE
Double checking Unique Visitor count vs Google Analytics definition

### DISCUSSION THREAD

**Discussion by oscarbaruffa** [plausible/analytics#536] [Anything Else]
Q: I have a lot of repeat visitors to my site, so I should expect to see much higher number of uniques when using Plausible vs GA. Is this correct?

I think I understand this, but I want to double check. 

GA's unique count will be unique number of visitors, over the time period selected, given whatever cookies etc remain in place in the visitor's browser. 

For Plausible, Unique is just counted for visits from a user in a single day. So someone visiting once per day will be counted at 7 users within a 7 day week.

**Comment by metmarkosaric** [comment]
Exactly @oscarbaruffa! In order to be GDPR compliant so you don't need to ask for consent from your visitors to track them, we cannot use any forms of persistent identifiers such as cookies.

This means that we're tracking all fine on the same day (if the same person visits your site 3 times on the same day we will show them as 1 unique visitor) but we cannot do any tracking across days (if the same person visits your site 3 times on 3 different days during the month we will show them as 3 unique visitors).

**Reply by oscarbaruffa** [reply]
Great, thanks :)
